src/PrestaShopBundle/Resources/views/Admin/macros.html.twig

```diff
@@ -122,7 +122,7 @@
   {{ form_widget(form, vars) }}
 
   {% if extraVars.help is defined %}
-    <small class="form-text">{{ extraVars.help }}</small>
+    <small class="form-text">{{ extraVars.help|raw }}</small>
```

Comment:
This is not secure. Why do you need this ?

Reply:
Removed |raw usage here and instead for now created separate div content here https://github.com/PrestaShop/PrestaShop/pull/12331/commits/e1a8eb53a8757e48011f2dc4d32347816fbddd44#diff-a9cb32b99ab7a41df75b67b58b8dccf8R145. Anyway, I'm using |raw here as well to not loose existing translation but since no variable is involved it should be okey? :thinking:

Reply:
Yes, no variable involved => we're secure 👍